README.md

```diff
@@ -1,15 +1,12 @@
-# nightfall_dlp
-![nightfalldlp](https://cdn.nightfall.ai/nightfall-dark-logo-tm.png "nightfalldlp")
-### NightfallDLP - a code review tool that protects you from committing sensitive information
-
-NightfallDLP scans your commits for secrets or sensitive information and posts review comments to your code hosting 
-services automatically. It’s intended to be used as a part of your CI to simplify the development process, improve your 
-security, and ensure you never accidentally leak secrets or other sensitive information via an accidental commit.
-
-## Supported CI Services
-### GithubActions
-Example using the NightfallDLP Github Action inside a Github Workflow  
-_(**N.B.** you must use the actions/checkout step before the running the nightfalldlp action in order for it to function properly)_
+# Nightfall DLP Action
+![nightfalldlp](https://nightfall.ai/wp-content/uploads/2020/08/nightfall-dark-logo-tm-e1597263930794.png)
+## [Nightfall](https://nightfall.ai) DLP Action: A code review tool that protects you from committing sensitive information to your repositories.
+
+The Nightfall DLP Action scans your code commits upon Pull Request for sensitive information - like credentials & secrets, PII, credit card numbers & more - and posts review comments to your code hosting service automatically. The Nightfall DLP Action is intended to be used as a part of your CI to simplify the development process, improve your security, and ensure you never accidentally leak secrets or other sensitive information via an accidental commit.
+
+## Example
+Here's an example using the Nightfall DLP Github Action inside a Github Workflow:  
+_**Note:** you must use the actions/checkout step as shown below before the running the nightfalldlp action in order for it to function properly_
 ```yaml
 name: nightfalldlp
```

Comment:
The example yaml still says `uses: nightfallai/nightfall_dlp_action@CreateAction`, do we want to update this to a version?